Formula/noti.rb

```diff
@@ -1,11 +1,10 @@
 require "language/go"
 
 class Noti < Formula
-  desc "Displays a notification after a terminal process finishes."
+  desc "Trigger notifications when a process completes"
   homepage "https://github.com/variadico/noti"
-  url "https://github.com/variadico/noti/archive/v2.1.1.tar.gz"
-  sha256 "c31c763eb92b4600371b2a62f6dc1dbb3390d89c09e0baab7af0dc427b55a8c5"
-  head "https://github.com/variadico/noti.git", :branch => "dev"
```

Comment:
It is. I keep the latest release on `master` and make sure that's stable, but I don't make any guarantees for `dev`. This could be pulling bad code.